tests/functional_tests/iterator_tests/test_tuple.py

```diff
@@ -198,6 +198,55 @@ def test_field_of_tuple_output(fencil_processor_no_gtfn_exec, stencil):
         assert np.allclose(inp2, out_np[:]["f1"])
 
 
+@pytest.mark.parametrize(
```

Comment:
Optional, but you could add a test for a tuple of tuples.

Reply:
done